src/db/file/repo.js

```diff
@@ -140,10 +156,9 @@ exports.deleteRepo = async (name) => {
   });
 };
 
-exports.isUserPushAllowed = async (name, user) => {
-  name = name.toLowerCase();
+exports.isUserPushAllowed = async (url, user) => {
   return new Promise(async (resolve, reject) => {
-    const repo = await exports.getRepo(name);
+    const repo = await exports.getRepoByUrl(url);
```

Comment:
Again, same point here - this is fine moving forward for updated data properties but what about previous stored data? Do we need a transformer?